cfgov/v1/util/util.py

```diff
@@ -79,9 +79,9 @@ def get_secondary_nav_items(request, current_page):
     nav_items = []
     for sibling in pages:
         if page.id == sibling.id:
-            sibling = page.get_appropriate_page_version(request)
```

Comment:
Hm, you're right, it does seem like those would have been the same. I really don't know.